### PR TITLE
Updating "Migrating an Existing Subgraph to The Graph Network" page

### PR DESCRIPTION
**Pull request by ahmadmardeni1**
- I've fixed some readability issues.
- Added "When not to Migrate?" section.
- Added Curation Station links.

### CODE REVIEW

File: pages/en/hosted-service/migrating-subgraph.mdx [hunk 1]
```diff
@@ -4,11 +4,18 @@ title: Migrating an Existing Subgraph to The Graph Network
...
 
-This will tell you everything you need to know about how to migrate to the decentralized network and manage your subgraphs moving forward. The process is quick and your subgraphs will forever benefit from the reliability and performance that you can only get on The Graph Network.
+The process of migration is quick and your subgraphs will forever benefit from the reliability and performance that you can only get on The Graph Network.
 
-### Migrating An Existing Subgraph to The Graph Network
+### When not to Migrate?
+
+If your subgraph:
+
+- Is not generating too many queries, and at the same time you are not willing to self-signal.
```
Comment (schmidsi) [inline pages/en/hosted-service/migrating-subgraph.mdx:15]: Good idea with this section. We should be careful about the wording here. Too little queries is not a reason to not migrate. Also self-signalling is not necessary a problem. But these are good reasons to not migrate:
- Usage of IPFS
- Usage of full-text search
- Deployed on other chains than Ethereum mainnet

File: pages/en/hosted-service/migrating-subgraph.mdx [hunk 2]
```diff
@@ -64,6 +71,8 @@ graph codegen && graph build
 
 And that's it! After you are done publishing, you'll be able to view your subgraphs live on the network via [The Graph Explorer](https://thegraph.com/explorer).
 
+Feel free to leverage the Curation Station (Unofficial) [Discord](https://discord.gg/GVfH3PgANH) and [Telegram](https://t.me/+RqzR_Y_a5eY2Zjdh) to let Curators know that your subgraph is ready to be signaled. It would also be helpful if you share your expected query volume with them.
```
Comment (schmidsi) [inline pages/en/hosted-service/migrating-subgraph.mdx:74]: These are unofficial sources and we should not reference them on the public docs. But a paragraph that explains the general process of advertising a subgraph is definitely good. Main place for such thing is the official The Graph Discord: thegraph.com/discord

**Review by schmidsi — CHANGES_REQUESTED** [review]
Thanks for your input. I added a few comments.

**Comment by ahmadmardeni1** [comment]
Done. @schmidsi
Also, changed the expired link:

Capture

File: pages/en/hosted-service/migrating-subgraph.mdx [hunk 3]
```diff
@@ -4,11 +4,19 @@ title: Migrating an Existing Subgraph to The Graph Network
...
+The process of migration is quick and your subgraphs will forever benefit from the reliability and performance that you can only get on The Graph Network.
 
-### Migrating An Existing Subgraph to The Graph Network
+### When not to Migrate?
+
+If your subgraph is:
+
+- Indexing [IPFS](https://ipfs.io/).
+- Using [full-text search fields](https://thegraph.com/docs/en/developer/create-subgraph-hosted/#defining-fulltext-search-fields).
+- Deployed on other chains than Ethereum mainnet.
```
Comment (azf20) [inline pages/en/hosted-service/migrating-subgraph.mdx:17]: "Indexing chains other than Ethereum mainnet" might be slightly clearer here, just to be consistent across the documentation (subgraphs are deployed to an instance of Graph Node).

File: pages/en/hosted-service/migrating-subgraph.mdx [hunk 4]
```diff
@@ -64,6 +72,8 @@ graph codegen && graph build
 
 And that's it! After you are done publishing, you'll be able to view your subgraphs live on the network via [The Graph Explorer](https://thegraph.com/explorer).
 
+Feel free to leverage the [#Curators channel](https://discord.gg/rC8rBuRtbH) on Discord to let Curators know that your subgraph is ready to be signaled. It would also be helpful if you share your expected query volume with them.
```
Comment (azf20) [inline pages/en/hosted-service/migrating-subgraph.mdx:75]: Nice! Maybe works best to clarify how this information helps (maybe "Help curators by giving them some context about your subgraph, and its expected query volume")

**Review by azf20 — CHANGES_REQUESTED** [review]
Thanks so much for this PR! Some small wording suggestions in line

**Comment by ahmadmardeni1** [comment]
Done. @azf20

**Review by azf20 — APPROVED** [review]

**Review by schmidsi — APPROVED** [review]

**Pull request merged**